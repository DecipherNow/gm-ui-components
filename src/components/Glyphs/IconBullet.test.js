import React from "react";
import { shallow } from "enzyme";

import IconBullet from "./IconBullet";

describe("IconBullet", () => {
  it("matches snapshot", () => {
    const aIconBullet = shallow(<IconBullet />);
    expect(aIconBullet).toMatchSnapshot();
  });
});