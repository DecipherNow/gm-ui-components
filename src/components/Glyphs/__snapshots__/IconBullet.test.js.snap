// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconBullet matches snapshot 1`] = `
<Icon
  backgroundColor="currentColor"
  backgroundOpacity={1}
  backgroundSizeRatio={1}
  borderColor="currentColor"
  borderOpacity={1}
  borderWidth={1}
  glyph=""
  glyphColor="currentColor"
  glyphName="Bullet"
  glyphSizeRatio={1}
>
  <circle
    cx="12"
    cy="12"
    fillRule="nonzero"
    r="5.00000002"
  />
</Icon>
`;
